### ISSUE TITLE
Training resnet50 to use with Relevance-CAM

### ISSUE THREAD

**Issue by juliadietlmeier** [mongeoroo/Relevance-CAM#6]
Hi @mongeoroo,

Thanks for the incredible work – the results in the paper look great!

I am looking to use Relevance-CAM with my custom dataset. I would need to train ResNet50 (initialized with ImageNet) on my custom dataset first. Can I use the resnet50 model from your repo?  Will I need to write the training script? How do I approach this? I think the Multi_CAM.py shows only the inference examples. Thank you!

**Comment by mongeoroo** [comment]
Thanks for your interest!

But Relevance-CAM is Explainable AI method that doesn't include training sessions. 

In our paper, we use ImageNet pretrained weight provided in pytorch.

If you want to train your classification network, you can train the model through well-known loss such as cross-entropy loss.

**Comment by juliadietlmeier** [comment]
Thank you for this clarification!
